### PR TITLE
Add support for ACSes in the content app

### PR DESCRIPTION
**Pull request by daviddavis**
fixes #8749

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

**Comment by daviddavis** [comment]
@pavelpicka @ipanova @bmbouter this should be ready for review now.

File: pulpcore/content/handler.py [hunk 1]
```diff
@@ -665,7 +665,7 @@ async def _stream_content_artifact(self, request, response, content_artifact):
         """
 
         def get_remote_artifacts_blocking():
-            return list(content_artifact.remoteartifact_set.all())
+            return list(content_artifact.remoteartifact_set.order_by_acs().all())
```
Comment (bmbouter) [inline pulpcore/content/handler.py:668]: Niiiiiice

**Review by bmbouter — APPROVED** [review]

File: pulpcore/content/handler.py (hunk 1 above)
Comment (ipanova) [reply pulpcore/content/handler.py:668]: :rocket:

**Review by ipanova — APPROVED** [review]

File: pulpcore/content/handler.py (hunk 1 above)
Comment (ipanova) [reply pulpcore/content/handler.py:668]: is all() necessary?

Comment (daviddavis) [reply pulpcore/content/handler.py:668]: Nope, I'll remove it.

**Pull request merged**